### PR TITLE
Add image, index, stylesheet and linters file

### PR DESCRIPTION
**Pull request by dfizzkruah93**
Added image file containing images
Added index and stylesheet
Added linters file

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  height: auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 0;
+  margin-top: 58px;
+}
```
Comment (keroloussamy) [inline styles.css]: - [ ] Kindly make the `margin-top: 58px;` smaller to like the Figma design.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+h1 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 60px;
+  margin-top: 100px;
+  vertical-align: 24px;
+}
```
Comment (keroloussamy) [inline styles.css]: - [ ] Kindly change the `font-weight: bold;` to something a litter bigger like `font-weight: 800;` to be like the design.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @dfizzkruah93, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (dfizzkruah93) [reply styles.css]: @keroloussamy . Kindly review changes made to the code as was stated.
This is  my last review and I want to make sure before sending it for final review

**Comment by dfizzkruah93** [comment]
@keroloussamy. Kindly disregard the first and see this changes made for review to the code as was stated.
I only have the  last review request  and I want to make sure before sending it for final review.

Thank you.

**Comment by keroloussamy** [comment]
@dfizzkruah93 Yes you can submit, You have implemented the changes.

**Comment by dfizzkruah93** [comment]
@keroloussamy Thanks very much

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @dfizzkruah93 ,  

Your project is complete! Great job for nailing it 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — APPROVED** [review]
### Hi @dfizzkruah93 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — COMMENTED** [review]
Hi @dfizzkruah93 , 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**